src/bundle/Resources/config/routing_rest.yaml

```diff
@@ -137,3 +137,14 @@ ibexa.rest.content_type.load_field_definitions_from_expression:
     methods: [POST]
     options:
         expose: true
+
+#
+# Site Access
+#
+
+ibexa.rest.site_access.load_for_location:
+    path: /site-access/load-non-admin-for-location/{locationId}
```

Comment:
Not gonna lie, this URI terrifies me 😅 

`load-non-admin-for-location`? Seems like extremely specific. It definitely violates the REST principle by putting some "action" (`load`) as part of the URI.

I'd suggest at the very least changing the part I mentioned.

If possible, I'd prefer this endpoint to be more generic. In my opinion ideal URI would look like this:
```
/site-access/by-location/{locationId}?resolver_type=non_admin
```
I would suggest to allow picking a specific type of resolver we're looking for by using a query parameter instead of creating separate links for each and every one (if/when they become needed).